airbyte-integrations/connectors/source-amplitude/source_amplitude/spec.yaml

```diff
@@ -0,0 +1,47 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/amplitude
+connectionSpecification:
+  "$schema": http://json-schema.org/draft-07/schema#
+  title: Amplitude Spec
+  type: object
+  required:
+  - api_key
+  - secret_key
+  - start_date
+  additionalProperties: true
+  properties:
+    data_region:
+      type: string
+      title: Data region
+      description: Amplitude data region server
+      enum:
+      - Standard Server
+      - EU Residency Server
+      default: Standard Server
+    api_key:
+      type: string
+      title: API Key
+      description: Amplitude API Key. See the <a href="https://docs.airbyte.com/integrations/sources/amplitude#setup-guide">setup
+        guide</a> for more information on how to obtain this key.
+      airbyte_secret: true
+    secret_key:
+      type: string
+      title: Secret Key
+      description: Amplitude Secret Key. See the <a href="https://docs.airbyte.com/integrations/sources/amplitude#setup-guide">setup
+        guide</a> for more information on how to obtain this key.
+      airbyte_secret: true
+    start_date:
+      type: string
+      title: Replication Start Date
+      pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
+      description: UTC date and time in the format 2021-01-25T00:00:00Z. Any data
+        before this date will not be replicated.
+      examples:
+      - "2021-01-25T00:00:00Z"
+    request_time_range:
```

Comment:
can you clarify why this is being added as part of this PR? I would have thought a migration PR strives to keep exactly the same external UX

Reply:
I've added `request_time_range` as part of https://github.com/airbytehq/airbyte/issues/21010, this is optional field.